### DISCUSSION TITLE
Как настроить ARI?

### DISCUSSION THREAD

**Discussion by i4j5** [mikopbx/Core#52] [Q&A] [answered]

**Comment by boffart** [comment, marked as answer]
Смотря какой функционал Вам нужен. Если работать со стазис, то вряд ли что то стоящее получится.
Все необходимые модуля в MIKOPBX для настройки ARI есть. Но если начнете править dialplan, то с большой вероятностью сломаете запись разговоров и легирование CDR.

**Reply by ogt111** [reply]
Вот чем-то схожая беда. Не хочется возвращаться на нативный астериск. А тут задачу поставили. Создал очередь для группы технической поддержки, а они еще и используют omnidesk, как таск борду. И затребовали интеграцию с ним. Там имеется интеграция с asterisk и там как раз нужен ARI и модификация dialplan.

https://support.omnidesk.ru/knowledge_base/item/248033?sid=4/#scroll_to_0

Вот и никак не могу задействовать ARI. Даже конфига нет. И как правильно модифицировать dialplan, ну никак не одолею. :-(
